app/views/layouts/_shim.html.erb

```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.min.js">
+  </script>
+<![endif]-->
```

Comment:
Can you explain why we are using this script

Reply:
use for if the version of Microsoft Internet Explorer (IE)
is less than 9.
it’s actually a conditional comment supported by Internet Explorer
browsers for just this sort of situation.